crates/red_knot_python_semantic/resources/mdtest/suppressions/no_type_check.md

```diff
@@ -64,6 +66,8 @@ from typing import no_type_check
 @no_type_check
 @unknown_decorator
 def test() -> int:
+    # TODO: this should not be an error
+    # error: [unresolved-reference]
```

Comment:
These are regressions, but after talking about it, we consider them fairly low priority. I will open a ticket to address them as soon as this PR is merged.

Reply:
https://github.com/astral-sh/ruff/issues/17142